CriticalMass/Preferences.swift

```diff
@@ -6,40 +6,20 @@
 //
 
 import Foundation
+import UIKit
```

Comment:
why do wee need an UIKit import now?

Reply:
Can go I think. 
Was there because of the defaults